bungee/src/main/java/me/confuser/banmanager/webenhancer/bungee/BungeePlugin.java

```diff
@@ -97,7 +97,7 @@ private PluginInfo setupConfigs() throws IOException {
 
   private void setupCommands() {
     for (CommonCommand cmd : plugin.getCommands()) {
-      new BungeeCommand(cmd, this);
+      new me.confuser.banmanager.webenhancer.bungee.BungeeCommand(cmd, this);
```

Comment:
Is this change needed?

Reply:
Oh, no thats my fault. Did an oopsie there